ecl/eclcc/eclcc.cpp

```diff
@@ -2809,7 +2809,7 @@ int EclCC::parseCommandLineOptions(int argc, const char* argv[])
         else if (iter.matchOption(optMonitorInterval, "--monitorinterval"))
         {
         }
-        else if (iter.matchOption(tempArg, "-main"))
+        else if (iter.matchOption(tempArg, "-main") || iter.matchOption(tempArg, "--main"))
```

Comment:
for compatibility with ecl run.  I will add a jira to make all options compatible.